### ISSUE TITLE
Page - PDS Birth Notification

### ISSUE THREAD

**Issue by james-answer** [nhsconnect/Demographic-Event-Messages#2]
The PDS Birth Notification page (https://developer.nhs.uk/apis/demographicupdates-beta/explore_pds_birth_notification.html) contains a number of bug and could do with some additional content.

1. This page could do with an introduction explaining when this message is going to be produced. A diagram showing the resources and how they are linked in the event message bundle would help the reader quickly visualize what to expect in the event.

2. The Data item requirements table is very long and the different resouces get mixed up when you read it. It would probably read easier if the section is broken down into population guidance for each of the resources, i.e. one table for mother patient resource, one for Organization resource, one for each observation, etc.

3. Level of detail is good but additional element population guidance would be helpful. For example in the name element it has detail that the "official" name should be populated but could do with additional guidance like this shall be the Spine recognised name. This is probably less important for this event as is it generated from PDS but for other events this level of detail will mean people populate the elements with the desired content, so it is probably a good idea to be consistent accross the specifications even if it is not specifically needed for this event.

4. BirthOrder, additional guidance that this is the position in the order of birth this person was born, 1st 2nd, 3rd, etc. rather than the number of births.

5. Birth place is an address so the rows don't make complete sense. It should probably specify the address elemnts which we are expecting to be populated.

6. This page would also benefit from an example of the JSON/XML the subscriber would recieve.

7. There should be details around specific values which have to be set, which relate to the common envent message structure such as "EMS-EventType-1" (https://fhir.nhs.uk/STU3/CodeSystem/EMS-EventType-1) which has to be set in the event header resource. Also what the focus element should point towards in the bundle as there are a number of different resources which could be the focus of the event message.

**Comment by james-answer** [comment]
I have started making changes to the structure of the page on a new branch "feature/birth_notification_changes".

There are a couple of requirements in the table which are not clear and may need clarification. Also I am going to try and add more description as to why each of the observations and other resources are included, also if they will always be included, for example will the still birth indicator observation always be included even if the baby is not still born as it is not clear.

I have noticed a lack of consistency between the different event messages as to which fields are populated for each resource type, which fields are mandatory, required or optional. We need to check that we are not adding complexity for the consumers of these event messages.
